### PR TITLE
Update blueprint dependencies and quarkus version to 2.16.2

### LINKED ISSUE #273: Support Quarkus 2.16.1.Final version
**Is your feature request related to a problem? Please describe.**
Update blueprint to support recently released Quarkus `v2.16.1.Final`

**Describe the solution you'd like**
Support latest version. Refer release notes: https://github.com/quarkusio/quarkus/releases/tag/2.16.1.Final

**Describe alternatives you've considered**
n/a

**Additional context**
n/a

### PR DESCRIPTION
**Pull request by vishal423**
Closes #273

### CODE REVIEW

File: generators/server/templates/quarkus/src/main/resources/application.properties.ejs [hunk 1]
```diff
@@ -98,6 +98,8 @@ quarkus.oidc.authentication.restore-path-after-redirect=false
 quarkus.oidc.roles.role-claim-path=roles
 quarkus.oidc.roles.source=userinfo
 quarkus.oidc.authentication.user-info-required=true
+quarkus.oidc.token.refresh-expired=true
+quarkus.oidc.token.refresh-token-time-skew=10S
```
Comment (mraible) [inline generators/server/templates/quarkus/src/main/resources/application.properties.ejs:102]: Just curious: why is this needed?

**Review by mraible — APPROVED** [review]

File: generators/server/templates/quarkus/src/main/resources/application.properties.ejs (hunk 1 above)
Comment (vishal423) [reply generators/server/templates/quarkus/src/main/resources/application.properties.ejs:102]: from docs:
image

Comment (mraible) [reply generators/server/templates/quarkus/src/main/resources/application.properties.ejs:102]: Thanks! Automatically refreshing tokens is a cool feature. 😎

**Review by mraible — APPROVED** [review]
LGTM!
